metadata-ingestion/src/datahub/ingestion/transformer/generic_aspect_transformer.py

```diff
@@ -0,0 +1,124 @@
+import logging
+from abc import ABCMeta, abstractmethod
+from typing import Iterable, Optional
+
+from datahub.emitter.mce_builder import Aspect
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.ingestion.api.common import EndOfStream, RecordEnvelope
+from datahub.ingestion.transformer.base_transformer import (
+    BaseTransformer,
+    SingleAspectTransformer,
+)
+from datahub.metadata.schema_classes import (
+    GenericAspectClass,
+    MetadataChangeEventClass,
+    MetadataChangeProposalClass,
+)
+from datahub.utilities.urns.urn import Urn
+
+log = logging.getLogger(__name__)
+
+
+class GenericAspectTransformer(
+    BaseTransformer, SingleAspectTransformer, metaclass=ABCMeta
+):
+    """Transformer that does transform custom aspects using GenericAspectClass."""
+
+    def __init__(self):
+        super().__init__()
+
+    def transform_aspect(
+        self, entity_urn: str, aspect_name: str, aspect: Optional[Aspect]
+    ) -> Optional[Aspect]:
+        """Do not implement."""
+        pass
```

Comment:
I can see for simplicity why you inherited from `SimpleAspectTransformer`, but I think it's awkward we're essentially ignoring the main method of that class.